packages/headless/src/controllers/commerce/core/sub-controller/headless-sub-controller.ts

```diff
@@ -67,10 +75,19 @@ export interface BaseSolutionTypeSubControllers {
    * @returns A `Pagination` sub-controller.
    */
   pagination(props?: PaginationProps): Pagination;
+
+  /**
+   * Builds a `Summary` controller.
```

Comment:
```suggestion
   * Creates a `Summary` sub-controller.
```

Reply:
Applied documentation changes in https://github.com/coveo/ui-kit/pull/4082/commits/a2b6110861ab7c44a5ebfc6a1429aabad825d4e1